app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/HomeFragment.java

```diff
@@ -192,4 +221,45 @@ public void onError(Throwable e) {
         }
 
     }
+
+    @Override
+    public void onCustomTabsConnected() {
+
+    }
+
+    @Override
+    public void onCustomTabsDisconnected() {
+
+    }
+
+    private void getTagline(){
+
+       Call<ArrayList<TaglineLanguageModel>> call = apiClient.getTagline();
+       call.enqueue(new Callback<ArrayList<TaglineLanguageModel>>() {
+           @Override
+           public void onResponse(Call<ArrayList<TaglineLanguageModel>> call, Response<ArrayList<TaglineLanguageModel>> response) {
+               if(response != null){
```

Comment:
You should check by **if (response.isSuccessful())** and handle it properly, otherwise the body might be null leading to NPE.